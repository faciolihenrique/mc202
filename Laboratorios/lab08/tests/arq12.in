# Teste com os alunos da turma E
#
i 154815 Breno Pedroni
i 160138 Caio Cesar de Campos Silva
i 043473 Fernanda Elena de Barros Reis
i 104923 Fernando Henrique de Moura Bastos
i 160160 Guilherme Furlan
i 146328 Guilherme Victor Cunha Câmara
i 155742 Hugo Ralf Pereira
i 155758 Igor Gustavo Hitzschky Lema
i 155925 Joao Pedro Brasao Toledo
i 155951 Joao Vitor Buscatto Silva
i 156074 Kaique de Souza Monteiro
i 156131 Lais Tiemi Murakami
i 117706 Lucas Cavalcante de Barros
i 156405 Lucas Rodolfo de Castro Moura
i 103179 Lucas Seiji Mizuta
i 105351 Maria Angelica de Araujo Barnabe
i 149332 Mateus Roque Buck
i 158372 Raphael Pavan Silva
i 157198 Rodrigo Dahlstrom Hilkner
i 160245 Rogerio Guimaraes Meirelles
i 157228 Ronaldo Prata Amorim
i 157240 Sabrina Beck Angelini
i 137796 Victor Otávio Andrade das Neves
i 139240 Vítor Monteiro Francisco dos Santos
i 158526 Yuri Catarino Delgado
#
p
n
a
c 155742
c 103179
d
l
x
