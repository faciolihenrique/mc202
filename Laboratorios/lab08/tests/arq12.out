Ação: # Teste com os alunos da turma E
Ação: #
Ação: i 154815 Breno Pedroni
Ação: i 160138 Caio Cesar de Campos Silva
Ação: i 043473 Fernanda Elena de Barros Reis
Ação: i 104923 Fernando Henrique de Moura Bastos
Ação: i 160160 Guilherme Furlan
Ação: i 146328 Guilherme Victor Cunha Câmara
